src/ur.rs

```diff
@@ -122,10 +134,44 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_ur_encoder_bc_crypto_request() {
+        // https://github.com/BlockchainCommons/crypto-commons/blob/master/Docs/ur-99-request-response.md
```

Comment:
Can we make this more stable by using 
https://github.com/BlockchainCommons/crypto-commons/blob/67ea252f4a7f295bb347cb046796d5b445b3ad3c/Docs/ur-99-request-response.md
?